spec/controllers/v2/categories_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe V2::CategoriesController, type: :controller do
+
+    describe 'GET #index' do 
+        let!(:categories) { 
```

Comment:
You could use https://github.com/ffaker/ffaker ffaker gem, that seeds data. 

Really cool tool for testing purposes.